releases/v21.1.0.md

```diff
@@ -0,0 +1,144 @@
+Core | **Import from Local Dump File** | The new [`cockroach import`](../v21.1/cockroach-import.html) command imports a database or table from a local `PGDUMP` or `MYSQLDUMP` file into a running cluster. This is useful for quick imports of 15MB or smaller. For larger imports, use the [`IMPORT`](../v21.1/import.html) statement.
+Core | **Additional Import Options** | New [`IMPORT` options](../v21.1/import.html#import-options) give you more control over the import process's behavior: The `row_limit` option limits the number of rows to import, which is useful for finding errors quickly before executing a more time- and resource-consuming import; the `ignore_unsupported_statements` option ignores SQL statements in `PGDUMP` files that are unsupported by CockroachDB; and the `log_ignored_statements` option logs unsupported statements to cloud storage or userfile storage when `ignore_unsupported_statements` is enabled.
+Core | **Re-validating Indexes During `RESTORE`** | Incremental backups created by v20.2.2 and prior v20.2.x releases or v20.1.4 and prior v20.1.x releases may include incomplete data for indexes that were in the process of being created. Therefore, when incremental backups taken by these versions are restored by v21.1.0, any indexes created during those incremental backups will be re-validated by [`RESTORE`](../v21.1/restore.html#restore-from-incremental-backups).
+
+### Database Operations
+
+Version | Feature | Description
+--------|---------|------------
+Core | **Logging Improvements** | Log events are now organized into [logging channels](../v21.1/logging-overview.html) that address different [use cases](../v21.1/logging-use-cases.html). Logging channels can be freely mapped to log sinks and routed to destinations outside CockroachDB (including external log collectors). All logging aspects, including message format (e.g., JSON), are now [configurable](../v21.1/configure-logs.html) via YAML.
+Core | **Cluster API v2.0** | This [new API](../v21.1/cluster-api.html) for monitoring clusters and nodes builds on prior endpoints, offering a consistent REST interface that's easier to use with your choice of tooling. The API offers a streamlined authentication process and developer-friendly [reference documentation](../api/cluster/v2.html).
+Core | **OpenShift-certified<br>Kubernetes Operator** | You can now [deploy CockroachDB on the Red Hat OpenShift platform](../v21.1/deploy-cockroachdb-with-kubernetes-openshift.html) using the latest OpenShift-certified Kubernetes Operator.  
+Core | **Automated TLS Cert Setup** | Using the new `cockroach connect` command and a secure token you provide, you can now automate the creation and distribution of a cluster's CA (certificate authority) and node certificates. Note that this feature is an alpha release with core functionality that may not meet your requirements.
```

Comment:
```suggestion
Core | **Auto TLS certificate setup** | [Using the new `cockroach connect` command](../v21.1/auto-tls.html), you can now let CockroachDB handle the creation and distribution among nodes of a cluster's CA (certificate authority) and node certificates. Note that this feature is an alpha release with core functionality that may not meet your requirements.
```

* Renamed to Auto TLS to match what we're using in the docs.
* Doc PR now merged - link to docs included.
* Removed mention of the token; probably too much detail, distracting from the core point.